From c0e8b2921395b331d55cb72dd23f1b63107e8403 Mon Sep 17 00:00:00 2001
From: Zheng Yang <zhengyang@rock-chips.com>
Date: Tue, 19 May 2015 17:53:55 +0800
Subject: [PATCH] HDMI: Not filter the input 3dmode with EDID information.

	If input 3dmode is not list in EDID, we print a
	warning log and send it to sink, no longer return
	error.

Signed-off-by: Zheng Yang <zhengyang@rock-chips.com>
---
 drivers/video/rockchip/hdmi/rockchip-hdmi-sysfs.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/video/rockchip/hdmi/rockchip-hdmi-sysfs.c b/drivers/video/rockchip/hdmi/rockchip-hdmi-sysfs.c
index 7d1b8e4370ac..95d60eb18ded 100644
--- a/drivers/video/rockchip/hdmi/rockchip-hdmi-sysfs.c
+++ b/drivers/video/rockchip/hdmi/rockchip-hdmi-sysfs.c
@@ -118,7 +118,7 @@ static int hdmi_set_3dmode(struct rk_display_device *device, int mode)
 
 	if ((mode != HDMI_3D_NONE) &&
 	    ((display_modelist->format_3d & (1 << mode)) == 0))
-		return -1;
+		pr_warn("warning: sink not support input 3d mode %d", mode);
 
 	if (hdmi->mode_3d != mode) {
 		hdmi->mode_3d = mode;
-- 
2.35.3

